examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/MainActivity.kt

```diff
@@ -35,6 +41,24 @@ class MainActivity : ComponentActivity(), PaywallResultHandler {
     }
 
     fun launchPaywall(offering: Offering? = null) {
-        paywallActivityLauncher.launch(offering)
+        paywallActivityLauncher.launch(
+            offering,
+            object : ParcelizableFontProvider {
+                override fun getFont(type: TypographyType): PaywallFontFamily? {
+                    return PaywallFontFamily(
+                        fonts = listOf(
+                            PaywallFont.GoogleFont(
+                                fontName = "Lobster Two",
+                                fontProvider = GoogleFontProvider(R.array.com_google_android_gms_fonts_certs),
+                            ),
+                            PaywallFont.ResourceFont(
+                                resourceId = R.font.super_mario,
+                                fontStyle = FontStyle.Italic,
```

Comment:
Note how by specifying the fontStyle to italic, this has more preference than the other font for italic, so we only use it there.